README.md

```diff
@@ -13,6 +13,22 @@ $ npm install
 ```
 
 
+## Starting Database
+
+Be sure to install MongoDB on your computer and start up the database using:
```

Comment:
Can you add a bit more detail on where/how to install mongodb? Maybe just links to https://docs.mongodb.com/manual/administration/install-on-linux/ and https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/